src/linter/SlangLinter.ts

```diff
@@ -101,7 +101,17 @@ export default class SlangLinter extends BaseLinter {
 
           let rex = line.match(re);
 
-          if (!docUri.endsWith(rex[1])) {
+          let errPath = rex[1];
+          if (isWindows) {
+            if (this.useWSL) {
+              errPath = this.convertToWslPath(errPath);
```

Comment:
`errPath` can cause misleading since the path is not limited to error but also note/warn. Something like `reportedFilepath` or `filePath` is better.

Reply:
I'm sorry for the delay. I also think `filePath` is good.
I'd like to rewrite the source code and commit it, but I'm not sure of the steps.
I apologize, but could you please instruct me on the steps?

Reply:
May i fix it?


maybe useful 
https://docs.github.com/ja/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork
https://docs.github.com/ja/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
@mshr-h san
I have modified Zoomer-k san code. I would like to incorporate the modified code into this pull-request, but I don't have permission. Could you please follow Zoomer-k's instruction url on how to change permissions and grant them to me?

 [https://docs.github.com/ja/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork](https://docs.github.com/ja/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

Reply:
@retar-kamuy 
It's already enabled by the PR author.
Seems like people with push access to the repo can also push the commit to the PR. (which means **I and the PR author** can push the commit but **you** can't)

Reply:
suggestion could commit to PR
https://bake0937.hatenablog.com/entry/2020/08/17/225229